### DISCUSSION TITLE
Run Time Error

### DISCUSSION THREAD

**Discussion by Resram** [KosinskiLab/AlphaPulldown#159] [Q&A]
Hello AlphaPulldown team,

  I installed the AlphaPulldown successfully. I tried to run but I received the following error: 

Loading module for CUDA 11.2.2
CUDA 11.2.2 is now loaded
2023-08-10 14:14:21.558753: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
FATAL Flags parsing error: flag --seq_index=: invalid literal for int() with base 10: ''
Pass --helpshort or --helpfull to see help on flags.

Any help would be greatly appreciated.
Thank you
Reshmi

**Comment by dingquanyu** [comment]
Hi,

Could you show us the full script of your command? It's very likely that in your script, there's something wrong after that ```seq_index``` parameter.

Cheers

**Reply by Resram** [reply]
Thank you. I could spot it. Resolved!
